templates/index.html

```diff
@@ -98,7 +106,7 @@ <h4 class="equal-height equal-height--mobile equal-height--align-middle">
 						<img src="{{ ASSET_SERVER_URL }}81ca0037-manage.svg" alt="Manage icon" class="no-margin">&nbsp;&nbsp;
 						<span>Manage</span>
 					</h4>
-					<p>Comes with a <a href="https://docs.ubuntu.com/maas/2.2/en/api" class="external">REST <abbr title="Application Programming Interface">API</abbr></a>, Web <abbr title="User Interface">UI</abbr> and <a href="https://docs.ubuntu.com/maas/2.2/en/manage-cli" class="external"><abbr title="Command Line Interface">CLI</abbr></a>.</p>
+					<p>Comes with a <a href="https://docs.ubuntu.com/maas/2.2/en/api" onClick="ga(‘send’, ‘event’, ‘Feature’, ‘Click’, ‘REST API, CLI’);" class="external">REST <abbr title="Application Programming Interface">API</abbr></a>, Web <abbr title="User Interface">UI</abbr> and <a href="https://docs.ubuntu.com/maas/2.2/en/manage-cli" onClick="ga(‘send’, ‘event’, ‘Feature’, ‘Click’, ‘REST API, CLI’);" class="external"><abbr title="Command Line Interface">CLI</abbr></a>.</p>
```

Comment:
Please give all links unique event values. Otherwise, they will be undistinguishable.